tasks/install-on-Debian.yml

```diff
@@ -1,4 +1,16 @@
 ---
+- name: Create the user's group
+  group:
+    name: "{{ buildkite_agent_username }}"
+    state: present
+
+- name: Create the user to run the agent
+  user:
+    name: "{{ buildkite_agent_username }}"
+    home: "/var/lib/{{ buildkite_agent_username }}"  # because the deb creates /var/lib/buildkite-agent by default.
+    uid: "{{ buildkite_agent_user_uid | default(omit) }}"
```

Comment:
https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#omitting-parameters

Reply:
And if this is omitted, then the user create should just - as per default behavior - just create the user with the next free uid?